CHANGES.rst

```diff
@@ -1,16 +1,31 @@
 Changes
 =======
 
+44.0 (2024-07-08)
+-----------------
+
+- Drop support for Python 3.8 (#315)
```

Comment:
question: any reason these don't align with past releases as well?

Reply:
:disappointed: Because copy-and-paste is an omnipresent opportunity for trivial excellence or repetitive failure. Fixing...

Reply:
This is fixed now. Thanks for catching this!